marvin.py

```diff
@@ -678,6 +688,20 @@ def error_handler(self, bot, update, error):
         """
         self.logger.warning('\nUpdate status:\n"%s"\nCaused error:\n"%s"', update, error)
 
+    def flooder_handler(self, bot, update):
+        """
+        User is flooding, take some action.
+        """
+        user_id = update.effective_user.id
+        self.logger.info("User is flooding: %s", user_id)
+        if self.antiflood_action == "none":
+            return
+        elif self.antiflood_action == "mute":
+            chat_id = update.message.chat.id
+            new_permissions = ChatPermissions(can_send_messages=False)
```

Comment:
In sostanza qui invece di inviargli l'oggetto c'è ancora la lista di booleani mi sembra di aver visto, prova a vedere, altrimenti se vuoi posso mettermi io a correggerlo